src/components/Root/Root.js

```diff
@@ -67,13 +67,15 @@ class Root extends Component {
     // * configure fetch and xhr interceptors to conduct RTR
     // * see SessionEventContainer for RTR handling
     if (this.props.config.useSecureTokens) {
-      const rtrConfig = configureRtr(this.props.config.rtr);
+      // FFetch relies on some of these properties, so we must ensure
+      // they are filled before initialization
+      this.props.config.rtr = configureRtr(this.props.config.rtr);
```

Comment:
Reassigning something on props feels icky. Why was it necessary to change this line?

Reply:
It's that, or reassigning the import (`configureRtr` doesn't change the input). This was changed because my FFetch imported config via `stripes-config`, whereas before it was passed in.

I can:
- revert it to being passed in instead, if preferred, but IMO it's best for the shared config to match the filled-in version from `configureRtr` (and we already do that in render(), where we have `gross: this overwrites whatever is currently stored at config.rtr`)
- make `configureRtr` mutate its input

Reply:
There are two things happening here with the RTR config: 
1. passing it to FFetch (in the constructor)
2. passing it Stripes (in render)

As you noted, we can get around (1) by updating FFetch to leverage stripes-config instead, and call `configureRtr()` in the constructor there. Should we just do the same in `render()` here? Part of me screams, "You should only have to call that function once! More than once means you're doing it wrong!" The other part of me is like, "Eh, if the value in stripes-config may be sparse, think of `configureRtr()` as `getSafeRtrValues()`. Does that still bug ya? No? Great."

Better, worse, just different but not actually better?

Reply:
Importing stripes-config for `render` here might make sense, but I'm not familiar enough with the rest of this to know if that's a bad idea.

What we can (and should) do is remove the `configureRtr` call inside `render()` here. It just populates the sparse values (as you said), and there's no reason any values should be removed between renders.